markup5ever/serialize.rs

```diff
@@ -6,35 +6,64 @@
 // <LICENSE-MIT or http://opensource.org/licenses/MIT>, at your
 // option. This file may not be copied, modified, or distributed
 // except according to those terms.
+//! Traits for serializing elements. The serializer expects the data to be xml-like (with a name,
+//! and optional children, attrs, text, comments, doctypes, and [processing instructions]). It uses
+//! the visitor pattern, where the serializer and the serializable objects are decoupled and
+//! implement their own traits.
+//!
+//! [processing instructions]: https://en.wikipedia.org/wiki/Processing_Instruction
 
 use QualName;
 use std::io;
 
 //§ serializing-html-fragments
+/// Used as a parameter to `serialize`, telling it if we want to skip the parent.
+/// TODO check: Currently the parameter to `ChildrenOnly` is unused.
```

Comment:
It's used by the HTML serializer code: https://github.com/derekdreery/html5ever/blob/30891d283147bfa815d1ae47694ff97761b2f150/html5ever/src/serialize/mod.rs#L77